dissect/target/plugins/os/windows/network.py

```diff
@@ -222,13 +224,32 @@ def _try_value(subkey: RegistryKey, value: str) -> str | list | None:
         return None
 
 
+def _get_config_value(key: RegistryKey, name: str) -> set:
+    value = _try_value(key, name)
+    if not value or value in ("", "0.0.0.0", None, [], ["0.0.0.0"]):
+        return set()
+
+    if isinstance(value, list):
+        return set(value)
+
+    return {value}
+
+
 class WindowsNetworkPlugin(NetworkPlugin):
+    """Windows network plugin."""
+
+    def __init__(self, target: Target):
+        super().__init__(target)
+        self._extract_network_device_config = lru_cache(4096)(self._extract_network_device_config)
+
     def _interfaces(self) -> Iterator[WindowsInterfaceRecord]:
+        """Yields found Windows interfaces used by :class:`NetworkPlugin` `interfaces` export func."""
```

Comment:
```suggestion
        """Yields found Windows interfaces used by :class:`NetworkPlugin` ``interfaces`` export func."""
```
Or is there an rst to refer directly to that method?